Express-API/express.js

```diff
@@ -77,28 +63,28 @@ app.get("/bands", cors(), function(request, response) {
   });
 });
 
-app.get("/allData", cors(), function(request, response) {
-  db.query(
-    "SELECT department.name AS 'DepartmentName', capability.name AS 'CapabilityName', role.name AS 'RoleName', role. role_id AS 'RoleId' FROM department JOIN capability ON department.department_id = capability.department_id JOIN role_capability ON role_capability.capability_id = capability.capability_id JOIN role ON role_capability.role_id = role.role_id JOIN band ON role.band_id = band.band_id WHERE department.department_id=" +
-      JSON.stringify(request.query.departmentID)
-  ).then(rows => {
-    response.send(rows);
-  });
+app.post('/login', cors(), function (request, response) {
+	console.log(request.body);
+	console.log(request.body.Password)
+	loginHandler.login(request.body, db).then(token => {
+    //console.log(token)
```

Comment:
If these logs are used purely for debugging they should be merged to master and should be removed.